Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -288,7 +300,7 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
         response: URLResponse?,
         error: Error?,
         completion: RequestCompletion?
-    ) {
+    ) async {
```

Comment:
I don't believe this function needs to be marked with `async`. There are no network calls happening in here, so it should be a synchronous function.